documentation/modules/cruise-control/proc-implementing-rebalance-proposal.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-implementing-rebalance-proposal-{context}']
+= Implementing a rebalance proposal
+
+If you are satisfied with a rebalance proposal, you can implement it by annotating the corresponding `KafkaClusterRebalance` resource. 
+
+At the next reconciliation, the Cluster Operator will instruct Cruise Control to rebalance the Kafka cluster. 
+Cruise Control then reassigns partitions to brokers as summarized in the rebalance proposal.
+
+WARNING: This is not a dry run. 
```

Comment:
Is _dry run_ universally understood?

Reply:
Yes, I think it's OK from a style perspective.